### ISSUE TITLE
Intrarenal Reflux

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6830]
**Preferred term label:**
Intrarenal reflux

**Synonyms**
IRR

**Definition (free text, please give PubMed ID)**
Severe vesicoureteral reflux reaching the kidney parenchyma.

PMID: 33738272, 18322164

**Parent term (use hpo.jax.org/app)**
Vesicoureteral reflux HP:0000076

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added under "High-grade VUR", please reopen if this is not good @RichardKlaus
